aiidalab_widgets_base/computational_resources.py

```diff
@@ -927,11 +927,20 @@ def on_setup_computer_success(self, function):
     def test(self, _=None):
         with self._test_out:
             clear_output()
-            print(
-                subprocess.check_output(
-                    ["verdi", "computer", "test", "--print-traceback", self.label.value]
-                ).decode("utf-8")
-            )
+            try:
+                print(
+                    subprocess.check_output(
+                        [
+                            "verdi",
+                            "computer",
+                            "test",
+                            "--print-traceback",
+                            self.label.value,
```

Comment:
Before running this command, can you please validate that the computer with this `label` exists?

Reply:
yep 👍